### ISSUE TITLE
[a11y-defect-1]: Information within user flow does not properly support all users.

### ISSUE THREAD

**Issue by ohgreatkate** [department-of-veterans-affairs/va.gov-team#64603]
### Point of contact

@ohgreatkate

### Severity level

1, Launchblocker. Critical. Must be fixed before launch.

### Details

...
When a Veteran decides to select 'finish this application later' they are directed to a confirmation screen validating that their information is saved. When the Veteran logs in at a later time goes to 'My VA' and reviews the section 'Benefit application drafts' this form is not there. 

If the Veteran navigates back to the benefits page and to apply their progress is saved.

### Reproduction steps

1. Log in
2. Start form application
3. select 'finish this application later'
4. Navigate to 'My VA' and look for the form..... but you won't find it.
...

### Proposed solution or next steps

...
Make sure in-progress applications that are saved for later populate in 'My VA'

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [ ] Axe-core
- [ ] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [X] Overall user experience
- [X] Cognitive | user flows & navigating
- [ ] URL standards
- [X] Something else

**Comment by zack** [comment]
@ohgreatkate Sorry if I'm missing this, but is this bug for a specific form? I'm seeing this behavior on 686C-674 and 21-526EZ. Is there another I should check?

**Comment by zack** [comment]
bump, @ohgreatkate any chance you cane take a look at this? thanks!
